src/main/java/org/zone/commands/structure/misc/Messages.java

```diff
@@ -26,47 +26,47 @@ private Messages() {
         throw new RuntimeException("Could not construct class");
     }
     //Universal Messages
-    public static Component setUniversalPlayerOnlyCommandErrorMessage() {
+    public static Component getUniversalPlayerOnlyCommandError() {
         return Component.text("Player only command").color(NamedTextColor.RED);
     }
 
-    public static Component setUniversalZoneSavingErrorMessage(ConfigurateException ce) {
+    public static Component getUniversalZoneSavingError(ConfigurateException ce) {
         return Component.text("Error when saving: " + ce.getMessage()).color(NamedTextColor.RED);
     }
 
-    public static Component setUniversalZoneRegionBuilderEnabledMessage() {
+    public static Component getUniversalZoneRegionBuilderEnabled() {
         return Component
                 .text("Region builder mode enabled. Run ")
                 .append(Component
                                 .text("'/zone create end'")
                                 .color(NamedTextColor.AQUA));
     }
 
-    public static Component setUniversalZoneRegionNameConflictErrorMessage() {
+    public static Component getUniversalZoneRegionNameConflictError() {
         return Component
                 .text("Cannot use that name")
                 .color(NamedTextColor.RED);
     }
 
-    public static Component setUniversalFlagCommandsRegionInteractMessageNoMessageSetErrorMessage() {
+    public static Component getUniversalFlagCommandsRegionInteractMessageNoMessageSetError() {
```

Comment:
I would split these. 

``getMessageTag()``

``getNoMessageSet()``

Reply:
what?

Reply:
If you split the two up. They can be combined later. This makes it more consistent